src/main/java/lotto/domain/winning/WinningStatistic.java

```diff
@@ -0,0 +1,91 @@
+package lotto.domain.winning;
+
+import java.text.DecimalFormat;
+import lotto.domain.lotto.PurchaseAmount;
+import lotto.domain.grade.Grade;
+import lotto.error.exception.NotExistGradeException;
+
+public class WinningStatistic {
+
+    private int firstGradeCount;
+
+    private int secondGradeCount;
+
+    private int thirdGradeCount;
+
+    private int forthGradeCount;
+
+    private PurchaseAmount purchaseAmount;
+
+    public WinningStatistic(PurchaseAmount purchaseAmount) {
+        this.purchaseAmount = purchaseAmount;
+    }
+
+    public int getFirstGradeCount() {
+        return firstGradeCount;
+    }
+
+    public int getSecondGradeCount() {
+        return secondGradeCount;
+    }
+
+    public int getThirdGradeCount() {
+        return thirdGradeCount;
+    }
+
+    public int getForthGradeCount() {
+        return forthGradeCount;
+    }
+
+    public int getGradeCount(Grade grade) {
+        if (grade == Grade.FIRST_GRADE)
+            return firstGradeCount;
```

Comment:
프로그래밍 요구사항에 있는 자바 코드 컨벤션을 따라, 중괄호를 생략하지 않고 작성해보면 좋을 것 같습니다.
https://myeonguni.tistory.com/1596
```suggestion
        if (grade == Grade.FIRST_GRADE) {
            return firstGradeCount;
        }
```

Reply:
코드 컨벤션 확인후 해당 피드백 수정해 놓을게요!